bitrise.yml

```diff
@@ -1,17 +1,28 @@
-format_version: 1.3.1
+format_version: 7
 default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
 
 app:
   envs:
-  - STEP_VERSION: 1.0.5
   # define these in your .bitrise.secrets.yml
-  - MY_PLIST_PATH: $MY_PLIST_PATH
+  - MY_PLIST_PATH: "ios-space-tester/a folder with spaces/BitriseSampleAppsiOS With Spaces/BitriseSampleAppsiOS With Spaces/Info.plist"
+  - SAMPLE_APP_REPOSITORY_URL: https://github.com/bitrise-samples/ios-space-tester
+  - SAMPLE_APP_CLONE_INTO_DIR: "./ios-space-tester"
+  
 
 workflows:
```

Comment:
please add the go test steps

Reply:
I see your point, but this is not a go step :)